interfaces/builtin/host_docs.go

```diff
@@ -0,0 +1,76 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2020 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/snapcore/snapd/interfaces"
+	"github.com/snapcore/snapd/interfaces/apparmor"
+	"github.com/snapcore/snapd/interfaces/mount"
+	"github.com/snapcore/snapd/osutil"
+)
+
+const hostDocsSummary = `allows access to documentation stored on the host`
```

Comment:
Should it be more specific about /usr/share/doc? I fear it may be unclear to users.

Reply:
Yes, I don't see this evolving into a generic interface that covers many docs, also one very quickly wonders why manpages are not covered, they are actually the main docs. /usr/shar/doc is mentioned in the FHS as such:

```
doc Miscellaneous documentation (optional)
```

https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch04s11.html#specificOptions15

"allows access to miscellaneous documentation from the system under /usr/share/doc" perhaps

Reply:
I tweaked this according to suggestions